### DISCUSSION TITLE
Please allow programs to take user input from the minibuffer (Windows port)

### DISCUSSION THREAD

**Discussion by aks2161989** [helix-editor/helix#9153] [General]
I am using the Windows port of Helix from [SourceForge](https://sourceforge.net/projects/helix-editor.mirror/). While I love the editor, I came across two problems when using it for coding:

1. There seems to be no placeholder for the current file. For instance, in Vim I could run the command `:!python %` and this would run the currently opened Python file in Vim. There seems to be n alternative to the '%' placeholder in Helix, so I need to run `run-shell-command python actual-file-name.py` which can be really inconvenient.
2. Also, the minibffer fails to run programs that take user input from the console. So it would be great to have a way to get user input, even if it opens a new cmd window

**Comment by kirawi** [comment]
1. https://github.com/helix-editor/helix/pull/6979
2. https://github.com/helix-editor/helix/issues/1976 (recommendation is to use a multiplexer)
